pkgs/tools/security/badchars/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+}:
+
+buildPythonApplication rec {
+  pname = "badchars";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname;
+    inherit version;
+    sha256 = "1xqki8qnfwl97d60xj69alyzwa1mnfbwki25j0vhvhb05varaxz2";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "argparse" ""
+  '';
+
+  # no tests are available and no module to install
+  doCheck = false;
```

Comment:
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "badchars" ];
```

Reply:
This would lead to an import error as there is no module.